maintainers/scripts/update.nix

```diff
@@ -55,7 +55,7 @@ let
 
           somewhatUniqueRepresentant =
             { package, attrPath }: {
-              inherit (package) updateScript;
+              updateScript = package.updateScript or (pkgs.nix-update-script { });
```

Comment:
Falling back to a single value is pointless – it is meant to distinguish different packages.